iac-extensions/arm/src/main/java/org/sonar/iac/arm/parser/bicep/TreeFactory.java

```diff
@@ -144,6 +144,10 @@ public Property objectProperty(Identifier key, SyntaxToken colon, Expression val
     return new PropertyImpl(key, colon, value);
   }
 
+  public Property objectProperty(InterpolatedString key, SyntaxToken colon, Expression value) {
+    return new PropertyImpl(key, colon, value);
+  }
+
```

Comment:
I was wondering if you could combine this method with the one above, as in `PropertyImpl `you only expect an `TextTree`.
So: `objectProperty(TextTree key, SyntaxToken colon, Expression value)`

Reply:
Damn that's pretty smart :thinking_face: I didn't push it hard enough with TextTree, I will definitely go this way!